supervisor/docker/manager.py

```diff
@@ -35,8 +35,8 @@
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
-MIN_SUPPORTED_DOCKER = "19.03.0"
-DOCKER_NETWORK_HOST = "host"
+MIN_SUPPORTED_DOCKER: Final = AwesomeVersion("20.10.0")
```

Comment:
```suggestion
MIN_SUPPORTED_DOCKER: Final = AwesomeVersion("20.10.1")
```

20.10.0 had an issue that would prevent auto start of the docker service